dump/create_pel.cpp

```diff
@@ -106,20 +109,18 @@ uint32_t createSbeErrorPEL(const std::string& event, const sbeError_t& sbeError,
                             "SLID={}", iter.first)
                     .c_str());
 
-            // Refer phosphor-logging/extensions/openpower-pels/README.md
-            // section "Self Boot Engine(SBE) First Failure Data Capture(FFDC)
-            // Support" for details of related to createPEL with SBE FFDC
-            // information usin g CreateWithFFDCFiles api.
-            pelFFDCInfo.emplace_back(
-                std::make_tuple(sdbusplus::xyz::openbmc_project::Logging::
-                                    server::Create::FFDCFormat::Custom,
-                                static_cast<uint8_t>(0xCB),
-                                static_cast<uint8_t>(0x01), iter.second.first));
+            auto tuple = iter.second;
```

Comment:
The comments are removed with the new changes. Is that intentional?

Reply:
yeah do not see that much need

Reply:
Let it be there